sled-agent/src/instance.rs

```diff
@@ -2496,38 +2489,6 @@ mod tests {
         }
     }
 
-    fn mock_vnic_contexts()
-    -> (MockDladmCreateVnicContext, MockDladmDeleteVnicContext) {
-        let create_vnic_ctx = MockDladm::create_vnic_context();
-        let delete_vnic_ctx = MockDladm::delete_vnic_context();
-        create_vnic_ctx.expect().return_once(
-            |physical_link: &Etherstub, _, _, _, _| {
-                assert_eq!(&physical_link.0, "mystub");
-                Ok(())
-            },
-        );
-        delete_vnic_ctx.expect().returning(|_| Ok(()));
-        (create_vnic_ctx, delete_vnic_ctx)
-    }
-
-    // InstanceManager::ensure_state calls Instance::put_state(Running),
-    //  which calls Instance::propolis_ensure,
```

Comment:
I'm working on a branch that renames a bunch of these routines and I'm grateful not to have to update any of the names here anymore ;)